src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -207,16 +207,25 @@ public function validate(mixed $value, Constraint $constraint): void
             }
         }
 
-        $errorPath = $constraint->errorPath ?? current($fields);
-        $invalidValue = $criteria[$errorPath] ?? $criteria[current($fields)];
+        $errorPath = $constraint->errorPath ?? implode(', ', array_keys($criteria));
+        $invalidValue = $criteria[$errorPath] ?? implode(', ', array_map(fn ($o) => is_object($o) ? '"object"' : $o, $criteria));
 
-        $this->context->buildViolation($constraint->message)
+        $violation = $this->context->buildViolation($constraint->message)
             ->atPath($errorPath)
             ->setParameter('{{ value }}', $this->formatWithIdentifiers($em, $class, $invalidValue))
             ->setInvalidValue($invalidValue)
             ->setCode(UniqueEntity::NOT_UNIQUE_ERROR)
-            ->setCause($result)
-            ->addViolation();
+            ->setCause($result);
+
+        if(is_array($criteria) && count($criteria) > 1)
+        {
+            foreach($criteria as $field => $value)
+            {
+                $violation->setParameter('{{ ' . $field . ' value }}', $this->formatWithIdentifiers($em, $class, $value));
+            }
+        }
```

Comment:
A few remarks to match Symfony CS:
- You should precede `is_array` and `count` with a backslash
- Opening brackets should be on the same line as `if` and `foreach`, and a space should be added after the keyword
- Concat operators shouldn't have spaces around it